pandas/core/reshape/merge.py

```diff
@@ -161,8 +161,9 @@ def merge_ordered(left, right, on=None,
         Interpolation method for data
     suffixes : 2-length sequence (tuple, list, ...)
         Suffix to apply to overlapping column names in the left and right
-        side, respectively. Except for tuple of (str, str), it also allows
-        tuple of (None, str) or (str, None)
+        side, respectively. And each element should be either a str or None,
```

Comment:
Sorry, this is still a bit confusing, and I don't think listing each combination really helps. How about.

```


suffixes : Sequence
    A length-2 sequence where each element is a optionally
    a string indicating the suffix to add to overlapping column
    names in `left` and `right` respectively. Pass a value of
    `None` instead of a string to indicate that the column names
    from `left` or `right` should be left as-is, with no suffix.
    At least one of the values must not be None.
```

Reply:
Thanks, Tom! and sorry for my bad docstring! and yours does look much clearer! I just removed a redundant 'a' ^^ @TomAugspurger